src/compiler/sys.ts

```diff
@@ -1135,7 +1155,7 @@ namespace ts {
 
         function getNodeSystem(): System {
             const nativePattern = /^native |^\([^)]+\)$|^(internal[\\/]|[a-zA-Z0-9_\s]+(\.js)?$)/;
-            const _fs: typeof import("fs") = require("fs");
+            const _fs: typeof import("fs") = wrapStacktraceReduction(require("fs"));

```

Comment:
if we are doing this in sys for all functions, we should do it for all the other uses of fs too.. (nodeTypingInstaller, server etc)

Reply:
Hm, fair enough, I just didn't realize other things used `fs` directly rather than `sys`.